tests/queries/0_stateless/02525_different_engines_in_temporary_tables.sql

```diff
@@ -0,0 +1,45 @@
+DROP TEMPORARY TABLE IF EXISTS table_merge_tree_02525;
```

Comment:
It cannot exist, because temporary tables are session-local, but okay

Reply:
As a prototype for my tests I considered:
tests/queries/0_stateless/00235_create_temporary_table_as.sql
tests/queries/0_stateless/00477_parsing_data_types.sql
tests/queries/0_stateless/00492_drop_temporary_table.sql

etc. All these tests call DROP TEMPORARY TABLE IF EXISTS  before create. 
If needed I may remove it. I added it because it's done so in other tests.